todoman/cli.py

```diff
@@ -313,8 +330,10 @@ def move(ctx, list, ids):
               'Defaults to true.')
 @click.option('--due', default=None, help='Only show tasks due in DUE hours',
               type=int)
+@click.option('--start', default=None, callback=_validate_start_date_param,
+              help='Only shows tasks after given DATE', type=str)
```

Comment:
I think `type=str` has no effect here, since we're already using a callback we can remove `type=str`.